packages/app/src/components/SearchBar/index.js

```diff
@@ -17,38 +18,54 @@ const SearchBar = () => {
   const [SearchResultData, setSearchResultData] = useContext(
     SearchResultDataContext
   );
-
-  const [NominationsData, setNominationsData] = useContext(
+  const [NominationsData] = useContext(
     NominationsDataContext
   );
+
+  const [counter, setCounter] = useState(0)
   const [showErrorMessage, setShowErrorMessage] = useState(false);
   const [startEmpty, setStartEmpty] = useState(true);
   const [modalIsOpen, setModalIsOpen] = useState(false);
 
+  function getNumOfAttachments(nominations) {
+    let counter = 0;
```

Comment:
Instead of creating a counter variable, this can be done this way: `nominations.filter(nomination => nomination.attachments).length`

Reply:
changed